modules/installation-proxy-configure-object.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * installing/insall_config/enable-cluster-wide-proxy.adoc
+
+[id="installation-proxy-configure-object_{context}"]
+= Enabling cluster-wide proxy
+
+The proxy object is used to manage cluster-wide egress proxy. When a cluster is
+installed or upgraded without proxy configured, a proxy object is still
+generated but it will have a nil spec and status. For example:
```

Comment:
s/spec/`spec`/
s/status/`status`/

Because of how the k8s API works, the `status` parameter provides the current state of the resource. I don't think a user can ever directly configure the `status` parameter, only the `spec` parameter. The `spec` parameter is where a user configures the expected state of the system, and `status` reports the current state.

So you may not need to mention anything about it specifically.

Also, in this example, the `spec` is not `nil`. It's a mapping that's basically:

```json
{
  "trustedCA": {
    "name": ""
  }
}
```
So the `name` parameter is set to the empty string.

Reply:
I agree, removing the "status" mention.